js_machine_front/src/components/About.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { memo } from 'react';

```

Comment:
please relocate all components related to routing into 'routes' folder

Reply:
fix